src/test/resources/mule_config/test-mule-proxy-config.xml

```diff
@@ -0,0 +1,70 @@
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:vault="http://www.mulesoft.org/schema/mule/vault" xmlns:tls="http://www.mulesoft.org/schema/mule/tls"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+          http://www.mulesoft.org/schema/mule/vault http://www.mulesoft.org/schema/mule/vault/current/mule-vault.xsd">
+
+
+    <vault:proxy name="gproxy" host="${proxyHost}" port="${proxyPort}" username="test" password="password" />
```

Comment:
Should this proxy config be referenced from the basicTrustStoreConfig?  This one currently isn't used from anywhere

Reply:
It's being referenced from the sharedTlsConfig